coreutils/yes.rs

```diff
@@ -1,16 +1,12 @@
+use crate::libbb::perror_nomsg_and_die::bb_perror_nomsg_and_die;
 use libc;
-use libc::putchar_unlocked;
-extern "C" {
-  #[no_mangle]
-  static mut stdout: *mut FILE;
+use libc::c_char;
+use libc::c_int;
+use std::ffi::CStr;
+use std::io;
+use std::io::Write;
+use std::slice;
 
-  #[no_mangle]
-  fn fputs_unlocked(__s: *const libc::c_char, __stream: *mut FILE) -> libc::c_int;
-  #[no_mangle]
-  fn bb_perror_nomsg_and_die() -> !;
-}
-
-use libc::FILE;
 /*
```

Comment:
Since we're basically rewriting the whole applet at this point I think we can get rid of these comments and docs. They're mostly used for the original busybox doc system which rustybox ignores.